src/js/angular/core/directives/queryeditor/templates/query-editor.html

```diff
@@ -219,7 +219,24 @@ <h6 class="alert alert-warning no-icon additional" ng-show="!queryIsRunning && !
                     <h6>{{'common.error' | translate}}<span ng-show="yasr.results.getException().status > 0"> {{yasr.results.getException().status}}: {{yasr.results.getException().statusText}}</span></h6>
                 </div>
 
-                <div class="plaintext">{{yasr.results.getException().responseText}}</div>
+                <div ng-if="!fullErrorMessage && sparqlQueryErrorInfo && sparqlQueryErrorInfo.shortenErrorMessage">
+                    <div class="show-full-message">
+                        <a href="#" ng-click="showFullErrorMessage(true)">
+                            <span class="show-full-message-btn">{{'query.editor.error.show.full.message' | translate}}</span>
```

Comment:
But it's a link actually.

Reply:
Also, why do we need this additional span around the link text?

Reply:
refactored